docs/DeveloperGuide.md

```diff
@@ -2,37 +2,207 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+Certain aspects of YMFC's code was inspired from the following member(s) IPs:
+
+[Sanjith](http://github.com/KSanjith/ip)
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Architecture
+![ArchitectureDiagram.png](img/ClassDiagrams/ArchitectureDiagram.png)
```

Comment:
Could considering including labels with your association arrows in the class diagrams